components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_lts.F

```diff
@@ -733,24 +711,30 @@ subroutine ocn_time_integrator_lts(domain,dt)!{{{
        do ie = 1, nEdgesInLTSRegion(1,2)
           iEdge = edgesInLTSRegion(1,2,ie)
           normalVelocityFirstStage(:,iEdge) = normalVelocityNew(:,iEdge)
-          normalVelocityTendSum1st(:,iEdge) = normalVelocityTendSum1st(:,iEdge) + normalVelocityTend(:,iEdge)
+          normalVelocityTendSum1st(:,iEdge) = ( 1.0_RKIND / (3.0_RKIND * dt * weightTendSum1st) * normalVelocityCur(:,iEdge) & 
+                                                + normalVelocityTendSum1st(:,iEdge) + normalVelocityTend(:,iEdge) ) &
+                                              * (1.0_RKIND - wettingVelocityFactor(:, iEdge))
```

Comment:
If `normalVelocityTend` comes from `ocn_tend_vel` then it has already been scaled by `(1.0_RKIND - wettingVelocityFactor(:, iEdge))`, but I'm a bit confused by what `normalVelocityTendSum1st` refers to since the first term does not appear to be a velocity tendency.

Reply:
or, I should say, it is used only for the slow terms

Reply:
@gcapodag Thanks for explaining. This makes sense to me.